drivers/display/display_renesas_lcdc.c

```diff
@@ -440,7 +440,7 @@ static void display_smartbond_get_capabilities(const struct device *dev,
 	 * bindings.
 	 */
 	capabilities->supported_pixel_formats = DT_INST_PROP(0, pixel_format);
-	capabilities->current_orientation = DISPLAY_ORIENTATION_NORMAL;
+	capabilities->current_orientation = DISPLAY_ORIENTATION_ROTATED_180;
```

Comment:
Is the display rotated 180 degrees or is this a work-around to rotate the input device coordinates?

Fyi there will be some changes regarding rotation and input device: https://github.com/zephyrproject-rtos/zephyr/pull/73474

Reply:
Thanks for the input. The display panel I am testing with should not be rotated. However, after the changes introduced in #70541 I was no longer able to make it work without playing with the `orientation` property.

Reply:
I tested with the changes proposed in #73474 and this solves the issues I was observing. As such, this PR should no longer be considered valid. @Snevzor can you confirm that this fix will be part of the upcoming LTS 3.7 release?

Reply:
Hi @ioannis-karachalios , that is up to @faxe1008 I presume. He is writing a unit test for this as well.

Reply:
OK. This should be considered a bug so I guess it should be included. I converted this PR to `Draft` until more info is available. Once again, thanks for notifying me.

Reply:
Look at this beauty: https://github.com/zephyrproject-rtos/zephyr/pull/73556